uwac/libuwac/uwac-window.c

```diff
@@ -641,6 +641,7 @@ UwacReturnCode UwacWindowSubmitBuffer(UwacWindow* window, bool copyContentForNex
 	if (window->pendingBuffer)
 	{
 		/* we already have a pending frame, don't do anything*/
+		UwacSubmitBufferPtr(window, window->pendingBuffer);
```

Comment:
That comment above should be changed ...

Reply:
done.